packages/ui/react-ui-grid/src/Grid/Grid.stories.tsx

```diff
@@ -50,34 +55,31 @@ const StoryGrid = ({ onEditingChange, ...props }: StoryGridProps) => {
     if (closestAccessory) {
       const action = closestAccessory.getAttribute('data-dx-grid-accessory');
       switch (action) {
-        case 'invoke-multiselect':
+        case 'invoke-multiselect': {
           triggerRef.current = closestAccessory as HTMLButtonElement;
           return setPopoverOpen(true);
+        }
       }
     }
   }, []);
 
-  const [initialCells, setInitialCells] = useState<GridContentProps['initialCells']>(storybookInitialCells());
-
-  const setMultiselectValue = useCallback((nextValue: string) => {
-    const nextInitialCells = storybookInitialCells(nextValue);
-    console.log('[set initial cells]', nextInitialCells);
-    setInitialCells(nextInitialCells);
-    setInternalMultiselectValue(nextValue);
-  }, []);
-
   return (
-    <>
-      <Grid.Root id='story' onEditingChange={onEditingChange}>
-        <Grid.Content {...props} initialCells={initialCells} onClick={handleClick} />
+    <div role='none' className='fixed inset-0 grid'>
+      <Grid.Root id='story' editing={editing} onEditingChange={handleEditingChange}>
+        {/* TODO(burdon): Why is this property not just "cells" or "values" */}
```

Comment:
Because this is React, and that would amount to a controlled state – this is qualified to signal that you don’t update values using this prop.

Reply:
Right, updates are done through a virtualization windowing API that isn’t as straightforward as React props for text inputs, for performance reasons.